gui/control_main.py

```diff
@@ -395,6 +395,7 @@ def createSampleTab(self):
         colStartLabel.setFixedWidth(140)
         colStartLabel.setAlignment(QtCore.Qt.AlignCenter)
         self.osc_start_ledit = QtWidgets.QLineEdit()
+        self.setGuiValues({"osc_start": "0.0"})
```

Comment:
were the fields of oscillation start, file number start, and resolution not shown when the request is not pre-filled?

Reply:
Correct, previously everytime the dewar tree refreshes, it would find out the current mounted sample and current request, then trigger a row_clicked programatically which would prefill the parameters. This would happen at GUI start too. Now since that doesn't happen on startup those values need a default